client/routes/search.js

```diff
@@ -45,6 +45,10 @@ module.exports = function (page) {
       var pageEl = document.getElementsByTagName('main')[0];
       pageEl.innerHTML = Templates['search'](ctx.state.data);
 
+      // Shows filter toggle button if javascript enabled
+      document.getElementById('fb').className = 'control__button';
```

Comment:
The button has the class `"control__button hidden"` to begin with. I want to remove the 'hidden' class